examples/deployment/assets/3-gold-delegated-asc.teal

```diff
@@ -2,10 +2,6 @@
 txn TypeEnum
 int 1
 ==
-txn Sender
-addr M7VR2MGHI35EG2NMYOF3X337636PIOFVSP2HNIFUKAG7WW6BDWDCA3E2DA
-==
-&&
 txn Amount
```

Comment:
Please don't delete from this file, this file is being used to show how delegated mode works in few examples

Reply:
This addr is deleted because we don't need this. In delegated mode the sender must be the account (or one of the accounts) that signed the logic signature (we discussed this). I have checked the example, that's unchanged. Adding a separate .teal file with similar logic for multisigned lsig seemed overkill.

Reply:
we need this address , it shows a working of specific example in `gold_sc.js` 
please check

Reply:
I have checked. It's fine. In future we can add a Receiver addr here and update `gold_sc.js` accordingly.